iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/procedure/impl/pipe/AbstractOperatePipeProcedureV2.java

```diff
@@ -192,14 +214,50 @@ protected TSStatus pushPipeMetaToDataNodes(ConfigNodeProcedureEnv env) throws IO
       pipeMetaBinaryList.add(pipeMeta.serialize());
     }
 
-    return RpcUtils.squashResponseStatusList(env.pushPipeMetaToDataNodes(pipeMetaBinaryList));
+    return env.pushPipeMetaToDataNodes(pipeMetaBinaryList);
+  }
+
+  /**
+   * Parsing the given pipe's or all pipes' pushPipeMeta exceptions to string.
+   *
+   * @param pipeName The given pipe's pipe name, null if report all pipes' exceptions.
+   * @param respMap The responseMap after pushing pipe meta
+   * @return Error messages for the given pipe after pushing pipe meta
+   */
+  protected String parsePushPipeMetaExceptionForPipe(
+      String pipeName, Map<Integer, TPushPipeMetaResp> respMap) {
+    final StringBuilder exceptionMessageBuilder = new StringBuilder();
+    for (Map.Entry<Integer, TPushPipeMetaResp> respEntry : respMap.entrySet()) {
+      int dataNodeId = respEntry.getKey();
+      TPushPipeMetaResp resp = respEntry.getValue();
+      if (resp.getStatus().getCode() == TSStatusCode.PIPE_PUSH_META_ERROR.getStatusCode()) {
+        exceptionMessageBuilder.append(String.format("DataNodeId: %s ", dataNodeId));
```

Comment:
if pipeName is given, and the result (pipeName.equals(message.getPipeName())) is always false, the returned value will not be empty, which may be a unexpeceted behaviour.